crates/extensions/tedge_log_manager/src/lib.rs

```diff
@@ -91,7 +91,7 @@ impl LogManagerBuilder {
         }
 
         // creating tedge-log-plugin.toml
-        let example_config = r#"# Add the list of log files that should be managed by tedge-log-plugin
+        let example_config = r#"# Add the list of log files that should be managed
```

Comment:
```suggestion
        let example_config = r#"# Add the list of log files that should be managed by tedge
```

Reply:
Though we're already in the context, e.g. it is under the /etc/tedge/plugins/*.toml files...so not sure if it is really required.